WIP: Rebuild now with a generated executive index

mhl opened the pull request:
The next step for this is to update the boundaries/index.json (possibly with the help of Louise's function for generating entries for a particular boundary directory) so that they now have the more specific position held IDs associated with boundaries correctly.

mhl committed:
Switch to an automatically generated executive/index.json

mhl committed:
Switch to an automatically generated legislative/index.json

alexdutton commented:
![](https://github.trello.services/images/mini-trello-icon.png) [CANADA: Clean executive index build](https://trello.com/c/2uzyllYH/1086-canada-clean-executive-index-build)

alexdutton committed:
Update position IDs for country and provinces

alexdutton committed:
Updated boundaries entries for cities

alexdutton committed:
Correct Wikidata ID for Sherbrooke.

The previous ID (Q3481908) was for a constituency dissolved in 1925,
whereas we want the extant one (Q3481910).

crowbot committed:
Refresh data: Automatically generated indexes

crowbot committed:
Rebuild data: Automatically generated indexes

crowbot committed:
Update commons-builder

crowbot committed:
Refresh data: updated commons-builder

crowbot committed:
Rebuild data: updated commons-builder

crowbot committed:
Regenerate executive index from wikidata

alexdutton committed:
Refresh data: regenerated executive index

alexdutton committed:
Rebuild data: regenerated executive index
